server/require.js

```diff
@@ -1,5 +1,5 @@
-import {join, parse, normalize, sep} from 'path'
-import fs from 'mz/fs'
```

Comment:
Can we remove `mz` with this change?

Reply:
No, it's still used, but there's another PR to remove it.